CHANGELOG.rst

```diff
@@ -1,13 +1,32 @@
 Changelog
 =========
 
-1.16.0 (WIP)
-------------
+1.16.0 (2024-11-25)
+-------------------
 
 **New features**
 
+* [#1733] Add experimental support for registering authentication context with mandates (machtiging)
+   and searching through them
+* [#1783] Add ``zaaktype__not_in`` filter to ``/zaken/api/v1/zaken/_zoek`` endpoint
+* [#1774] Add ``trefwoorden__overlap`` filter to ``/documenten/api/v1/enkelvoudiginformatieobjecten`` endpoint
+* [#1776] Support searching zaken on ``zaaktype.omschrijving`` in the Admin
+* [#1611] Add importing authorization data via yaml files as a configuration step
 * [#1817] Updated OAF version to 0.9.0. This upgrade allows admin users managing their sessions through the admin.
```

Comment:
I think maybe we should separate the library update into Project maintenance. 

Although I did approve the other PR :/